src/client/components/Header/__snapshots__/index.test.js.snap

```diff
@@ -39,13 +38,16 @@ exports[`<Header /> renders the header component 1`] = `
       >
         <img
           alt="iconAltText.home"
+          className="icon"
           height="15"
           src="images/home.svg"
           width="15"
         />
          
-        <span>
-          Home
+        <span
+          className="text"
+        >
+          header.home
```

Comment:
hm seems like this didn't render the translation in the snapshot, isn't it supposed to?

Reply:
interesting! it looks like all of our snapshot tests are broken in this way. i'm going to merge this PR and fix that issue separately.